types/stylelint/index.d.ts

```diff
@@ -203,6 +207,9 @@ declare namespace stylelint {
 		optional?: boolean;
 	};
 
+	/** @internal */
+	type RuleSeverity = Severity | RuleSeverityFunc;
```

Comment:
The type is hoisted I suppose but wouldn't it be clearer if it was set before line 145?